net/mptcp/mptcp_sched.c

```diff
@@ -88,11 +116,28 @@ static bool mptcp_is_available(struct sock *sk, const struct sk_buff *skb,
 	 */
 	if (skb && !zero_wnd_test &&
 	    after(tp->write_seq + min(skb->len, mss_now), tcp_wnd_end(tp)))
+		return true;
+
+	return false;
+}
+
+/* If the sub-socket sk available to send the skb? */
+static bool mptcp_is_available(struct sock *sk, const struct sk_buff *skb,
+			       bool zero_wnd_test)
+{
+	if (mptcp_is_def_unavailable(sk) ||
+	    mptcp_is_temp_unavailable(sk, skb, zero_wnd_test)) {
 		return false;
+	}
```

Comment:
Here you can also simply do:
return !mptcp_is_def_unavailable() && !mptcp_is_temp_unavailable();